src/libs/E2E/apiMocks/openReport.json

```diff
@@ -0,0 +1,115 @@
+{
+  "onyxData": [
+    {
+      "onyxMethod": "merge",
+      "key": "report_98345625",
+      "value": {
+        "reportID": "98345625",
+        "reportName": "Chat Report",
+        "chatType": "",
+        "ownerEmail": "__fake__",
+        "policyID": "_FAKE_",
+        "maxSequenceNumber": 1,
+        "participants": [
+          "alazutkinutest+0214@gmail.com",
```

Comment:
Is it OK to have all these emails listed in our open-source repo? Maybe we can replace them all with fake emails?

Reply:
good call! removed all personal details and stripped down the json